src/cache_entry.h

```diff
@@ -35,6 +35,9 @@
 
 namespace triton { namespace core {
 
+// Part of std in C++17
```

Comment:
May want to clarify this comment. Does this mean something along the lines of TODO: Change to std::byte when upgrade to C++17?

Reply:
done